api/Clipboard.json

```
@@ -187,34 +178,13 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": [
-              {
-                "version_added": "87",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.events.asyncClipboard.clipboardItem",
-                    "value_to_set": "true"
-                  }
-                ],
-                "notes": "Writing to the clipboard is available without permission in secure contexts and browser extensions, but only from user-initiated event callbacks. Browser extensions with the <code>\"clipboardWrite\"</code> permission can write to the clipboard at any time."
-              },
-              {
-                "version_added": "63",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.events.asyncClipboard.dataTransfer",
-                    "value_to_set": "true"
-                  }
-                ],
-                "partial_implementation": true,
-                "notes": [
-                  "This method accepts a <code>DataTransfer</code> object instead of an array of <code>ClipboardItem</code> objects.",
-                  "Writing to the clipboard is available without permission in secure contexts and browser extensions, but only from user-initiated event callbacks. Browser extensions with the <code>\"clipboardWrite\"</code> permission can write to the clipboard at any time."
-                ]
-              }
-            ],
+            "firefox": {
+              "version_added": "preview",
+              "notes": [
+                "Writing to the clipboard is available without permission in secure contexts and browser extensions, but only when the contexts have <a href='https://developer.mozilla.org/docs/Web/Security/User_activation#transient_activation'>transient activation</a>. Browser extensions with the <code>\"clipboardWrite\"</code> permission can write to the clipboard at any time.",
```

This is wrong because permission is not required anymore (or in spec). I am pretty sure this note reflects the spec outside of extensions - secure context and transient activation required and so that bit isn't needed.

For extensions I'm not clear on whether a "specification" exists, so impossible to know whether this is off-spec and hence needed. My assumption is that browser extensions have the same constraints as normal pages (transient activation and secure context) but you don't need either of those if you have the permission. I am querying but trying to get this as close as possible to correct now.

```suggestion
                "Browser extensions with the <code>\"clipboardWrite\"</code> permission can write to the clipboard at any time.",
```